src/main/java/com/backend/controller/PortfolioController.java

```diff
@@ -98,27 +98,33 @@ public CreatePortfolioAssetResponse addPortfolioAsset(@RequestBody CreatePortfol
 		if (request.getPortfolioId() == null) {
 			throw new BadRequestException(Constants.MESSAGE_INVALIDPORTFOLIOID);
 		}
-		if (request.getAssetId() == null) {
+		if (request.getAssetTicker() == null) {
 			throw new BadRequestException(Constants.MESSAGE_INVALIDASSETID);
```

Comment:
Should we change the constant error message as well for `MESSAGE_INVALIDASSETID`?